libcaf_net/test/datagram_transport.cpp

```diff
@@ -227,10 +225,9 @@ CAF_TEST(resolve and proxy communication) {
   using transport_type = datagram_transport<dummy_application_factory>;
   buffer_type recv_buf(1024);
   auto uri = unbox(make_uri("test:/id/42"));
-  auto mgr = make_endpoint_manager(mpx, sys,
-                                   transport_type{send_socket,
-                                                  dummy_application_factory{
-                                                    shared_buf}});
+  auto mgr = make_endpoint_manager(
```

Comment:
This formatting looks off to me. Can you check that we are using the same `.clang-format` as in the main repository?

Reply:
Indeed it does. I checked, but the `.clang-format` file is equal to the one from the main repo.